staging/publishing/rules.yaml

```diff
@@ -1612,3 +1612,12 @@ rules:
     name: master
     go: 1.13.9
 
+- destination: mount-utils
+  library: true
+  branches:
+  - source:
+      branch: master
+      dir: staging/src/k8s.io/mount-utils
+    name: master
+    go: 1.13.9
```

Comment:
Is this inconsistency with https://github.com/kubernetes/kubernetes/pull/91596/files#diff-51e587fd1a2b9eabc3950b4864ec5044R5 going to cause issues? cc @nikhita

Reply:
I can move to 1.14 and I don't see specific issues if I do. WDYT?

Reply:
Sure, I'm just not sure what the right thing to do is because nothing else here uses 1.14.

Reply:
It should either be `1.14.4` or be removed entirely. The go version for the master branch is unfortunately hardcoded in the bot today, but I'm going to work on fixing that this week/early next week - https://github.com/kubernetes/publishing-bot/issues/229 :)

Reply:
Can you also request a repo for mount-utils by creating a issue here - https://github.com/kubernetes/org/issues?

Reply:
@nikhita Thanks for the review. I would remove the go version if that is ok. 
We had the issue opened before that you addressed: https://github.com/kubernetes/org/issues/1887

Reply:
I think we need to have the explicit go version. It's currently failing the dependency pull job with go1.13

Reply:
The pull dependency job doesn't look at the go version in this file. I think that job just needs a run of `hack/update-vendor.sh`. You'll need to have go 1.14.4 locally while running that script to ensure that correct things are generated.

Reply:
>  We had the issue opened before that you addressed: kubernetes/org#1887

so many things going on, can't keep track anymore haha :smile:

Reply:
Ah sorry, I misread that! Looks like `go.mod` files in the staging repo still take 1.13 - https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/api/go.mod#L5

That looks like a bug, but I think it should be fine to manually update the go.mod file in mount-utils to go 1.13. The bug fix can go in post code freeze.